src/nupic/frameworks/opf/common_models/cluster_params.py

```diff
@@ -0,0 +1,139 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2015, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+import csv
+import json
+import numpy as np
+import os
+from pkg_resources import resource_stream
+
+
+def setRandomEncoderParams(params, minVal, maxVal, minResolution):
+  """
+  Given model params, figure out the correct parameters for the
+  RandomDistributed encoder. Modifies params in place.
+  """
+  encodersDict= (
+    params['modelConfig']['modelParams']['sensorParams']['encoders']
```

Comment:
I mis-typed.  It should be `"`, as described in the linked style guide.  I've since changed the original comment.